site/pages/api/create-consequence-services.api.ts

```diff
@@ -104,6 +105,18 @@ const createConsequenceServices = async (req: NextApiRequest, res: NextApiRespon
                 ? DISRUPTION_DETAIL_PAGE_PATH
                 : REVIEW_DISRUPTION_PAGE_PATH;
 
+        if (addAnotherConsequence) {
+            redirectToWithQueryParams(
+                req,
+                res,
+                template ? ["template"] : [],
+                `${TYPE_OF_CONSEQUENCE_PAGE_PATH}/${validatedBody.data.disruptionId}/${
+                    validatedBody.data.consequenceIndex + 1
```

Comment:
The highest index should be identified for increment. In case a user decides to use Change link in review disruption page and updates the first consequence and choses to add another then the index will be 2. If this disruption before editing already had more than one disruption this would result in an error

Reply:
done